config/config.go

```diff
@@ -137,6 +137,9 @@ func set() *Config {
 
 	v.SetDefault("NEWRELIC_EU_BASE_URL", "https://insights-collector.eu01.nr-data.net/v1/")
 
+	v.SetDefault("LOGS_LOGMESSAGE", false)
+	v.SetDefault("LOGS_HTTP", false)
```

Comment:
Why are these variables added? I don't them used in other place in the code

Reply:
Sorry - I'm not sure how this slipped through. These default values are part of the AddNRLogs branch. These won't cause any issues as they are not referenced anywhere else in this version of the nozzle, but I'm happy to submit another PR that doesn't include these two default values if you like.